lib/graphql/schema.ts

```diff
@@ -367,6 +367,7 @@ export type Mutation = {
   changeCardStatus: Card,
   changeCardPIN: ConfirmationRequest,
   confirmChangeCardPIN: ConfirmationStatus,
+  replaceCard: Card,
```

Comment:
Did you change the schema manually? We have a specific procedure to update it automatically from backend, sorry that it isn't documented, you can:
1) in backend, spin it up and comment in `services/backendService/src/core/graphql/server.ts`:
```
export const createGraphQLServer = async () => {
  const graphQLServer = express.Router();
  // graphQLServer.use(middlewares.getAuth());
  // graphQLServer.use(middlewares.extendRequestWithUser);
  graphQLServer.use(
    graphqlHTTP({
      schema: await createGraphQLSchema()
    })
  );
  return graphQLServer;
};
```
(this is to allow access to the schema without being authenticated)

2) in sdk run: `npm run schema:generate` 

3) commit corresponding changes to the schema

For example, when I do it, I can see that we are missing:
```
export type MutationReplaceCardArgs = {
  id: Scalars['String']
};
```

There are also some added stuff for `reorderCard`, you can also commit them if you like or leave them for the PR who will include the helper for that.

Reply:
Also, a good remark from @johannespfeiffer once the schema is updated with this new type (`MutationReplaceCardArgs`) as described in my comment above, it would make sense to use that directly as the type for the `replace` method (e.g. `async replace(args: MutationReplaceCardArgs) { ... }`) and get rid of the `ReplaceCardOptions` type in `lib/graphql/types` entirely.

I know that you followed the pattern used for other card helpers but as @johannespfeiffer correctly pointed out, it was a mistake from us

Please let me know if this is not clear 🙂